src/Airship.Net/IAirshipMessageCenter.cs

```diff
@@ -0,0 +1,62 @@
+/* Copyright Airship and Contributors */
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using AirshipDotNet.MessageCenter;
+
+namespace AirshipDotNet
+{
+    /// <summary>
+    /// Airship Message Center interface.
+    /// </summary>
+    public interface IAirshipMessageCenter
+    {
+        /// <summary>
+        /// Gets all messages from the message center.
+        /// </summary>
+        /// <returns>List of messages.</returns>
+        Task<List<Message>> GetMessagesAsync();
+
+        /// <summary>
+        /// Gets a specific message by ID.
+        /// </summary>
+        /// <param name="messageId">The message ID.</param>
+        /// <returns>The message or null if not found.</returns>
+        Task<Message?> GetMessageAsync(string messageId);
```

Comment:
Do we need it to be a task for it to be async? Can we drop the async parts?

Reply:
Yeah this is a basic pattern for async methods in C# - it's not the same as Swift. By "drop the async parts" are you referring to the naming?

Reply:
My misunderstanding. Lets drop the `Async` suffix though